jsonschema-core.xml

```diff
@@ -356,6 +356,11 @@
                     This extended meta-schema SHOULD be referenced using the "$schema" keyword, to
                     allow tools to follow the correct behaviour.
                 </t>
+                <t>
+                    Note that the recursive nature of meta-schemas requires re-definining
+                    recursive keywords in the extended meta-schema, as can be seen in
+                    ith the JSON Hyper-Schema meta-schema.
```

Comment:
typo: "ith"

Reply:
Thanks!  I'll fix and then merge, as I don't think this fundamentally changes the approval and this has no impact on implementations at all.